src/main/java/project/bookstore/repository/role/RoleRepo.java

```diff
@@ -0,0 +1,9 @@
+package project.bookstore.repository.role;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import project.bookstore.model.Role;
+import project.bookstore.model.RoleName;
+
+public interface RoleRepo extends JpaRepository<Role, Integer> {
```

Comment:
```suggestion
public interface RoleRepository extends JpaRepository<Role, Long> {
```
Don't use abbreviations in the names of variables, classes, and methods.
Also, you used Long as an id variable type.